bazel/envoy_build_system.bzl

```diff
@@ -213,6 +214,7 @@ def envoy_cc_test(name,
             ":" + name + "_lib",
             repository + "//test:main"
         ],
+        args = args + ["--gmock_default_mock_behavior=2"],
```

Comment:
It's hard to find the meaning of "2" here, because "gmock_default_mock_behavior" doesn't appear anywhere in the googletest codebase (it's defined with a special macro). Can you copy the meanings from that flag's --help into here as a comment?